.changeset/sixty-carpets-cheat.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
+
+Add Generate Models button to enable autogeneration of classes, associations, and mapping feature for relational databases
```

Comment:
lets separate this to two changeste files

The above message can stay for `@finos/legend-application-studio` but should be a `minor` change. 
For the `@finos/legend-graph` we can keep empty.